miner/worker.go

```diff
@@ -962,7 +969,14 @@ func (w *worker) commitTransaction(tx *types.Transaction, coinbase common.Addres
 	return receipt.Logs, traces, nil
 }
 
-func (w *worker) commitTransactions(txs types.OrderedTransactionSet, coinbase common.Address, interrupt *int32) (bool, bool) {
+func (w *worker) commitTransactions(txs types.OrderedTransactionSet, coinbase common.Address, interrupt *int32) (r1 bool, r2 bool) {
+	withTimer(l2CommitBlockTimer, func() {
+		r1, r2 = w.doCommitTransactions(txs, coinbase, interrupt)
+	})
+	return r1, r2
+}
+
+func (w *worker) doCommitTransactions(txs types.OrderedTransactionSet, coinbase common.Address, interrupt *int32) (bool, bool) {
```

Comment:
Let's do this instead of introducing a new function:

```go

func (w *worker) commitTransactions(txs types.OrderedTransactionSet, coinbase common.Address, interrupt *int32) (r1 bool, r2 bool) {
    defer func(t0 time.Time) {
        l2CommitBlockTimer.Update(time.Since(t0))
    }(time.Now())
```

Reply:
sure